### ISSUE TITLE
It seems not working on Mac OS

### ISSUE THREAD

**Issue by xiaohk** [AdrianLC/sublime-text-virtualenv#10]
I have tried the latest version on Mac OS, but I still cannot build my project properly using virtualenv. In my virtualenv I used Python 3.5, but when I used `python + virtualenv` to build, it used Python 2.9. 

When I do `virtualenv : activate`, I didn't get anything and the building still does not work.

Any suggestions?

**Comment by nervouna** [comment]
Works fine for me.

macOS 10.12.4 (16E195) / Sublime Text 3 Build 3126

**Comment by jakubLangr** [comment]
yes also does not seem to be working for me python 3.6, windows 10.

**Comment by a0g83agbc84** [comment]
For me neither... any workaround?
